apps/web/getSubdomainRegExp.js

```diff
@@ -0,0 +1,15 @@
+const getDefaultSubdomain = (url) => {
+  if (!url.startsWith("http:") && !url.startsWith("https:")) {
+    // Make it a valid URL. Mabe we can simply return null and opt-out from orgs support till the use a URL scheme.
+    url = `https://${url}`;
+  }
+  const _url = new URL(url);
+  const regex = new RegExp(/^([a-z]+\:\/{2})?((?<subdomain>[\w-.]+)\.[\w-]+\.\w+)$/);
```

Comment:
Only this regex changed with inclusion of dot in subdomain capturing group

Reply:
so, when the subdomain is org.app.cal.com the slug will be org.app?

Reply:
`subdomain` group would be org.app but `orgSlug` group would be **org**. This is because we are simply doing negative lookahead for `subdomain`.